ui/admin/app/components/form/alias/index.hbs

```diff
@@ -24,7 +24,7 @@
     {{#if @model.errors.name}}
       <F.Error as |E|>
         {{#each @model.errors.name as |error|}}
-          <E.Message>{{error.message}}</E.Message>
+          <E.Message data-test-error-message-name>{{error.message}}</E.Message>
```

Comment:
I see we add this `data-test-` selector, and this is all good and fine. But I do not see any selector using it. If we are not using this selector, do we need to add it?

Reply:
That error message is currently selecting on [an hds class](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/aliases/create-test.js#L29) and we agreed to move away from selecting on hds classes, so I removed `FIELD_ERROR_TEXT_SELECTOR` and replaced it with our common selector [FIELD_NAME_ERROR](https://github.com/hashicorp/boundary-ui/blob/750cc1ecae338621bdbfa3b0bac5dbac5d426d05/ui/admin/tests/helpers/selectors.js#L12), but I had to add the data test attr to that error for [this test](https://github.com/hashicorp/boundary-ui/blob/2e7c5716c2e1d712af9a17e94628d38e3ec534d6/ui/admin/tests/acceptance/aliases/create-test.js#L115).

Reply:
Gotcha. I miss that.

Recap. We are adding the data-test-* attribute to the template, so the common selector can be used within this creation form 😉